src/environments/environment.ts

```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  rmApiBase: 'https://rickandmortyapi.com/api/'
+  rmApiBase: 'https://rickandmortyapi.com/graphql'
 };
 
 /*
```

Comment:
Delete empty space and useless comment.

Reply:
this is the default information of  angular cli